addons/forum-post-countdown/addon.json

```
@@ -0,0 +1,38 @@
+{
+  "name": "Countdown for posting in the forums",
+  "description": "When posting in the forums, show a countdown on every page showing when you can post again.",
```

Here's my way of wording the name and description which you can apply and/or modify if you want:
```suggestion
  "name": "Display cooldown for posting in the forums",
  "description": "After submitting a forum post, display a countdown showing when you can post again.",
```

---

> Use nouns instead of verbs (for example, `Customizable new sprite position` instead of `Change new sprite position`) unless doing so would result in the addon name being overly complicated. (https://scratchaddons.com/docs/reference/addon-manifest/)

Because of that, I think cooldown and display should be flipped (so that display is the noun):

```suggestion
  "name": "Cooldown display for posting in the forums",
  "description": "After submitting a forum post, display a countdown showing when you can post again.",
```

Other than that, this seems like a better way to phrase it (I am not good with making names, as you can tell)